### PR TITLE
update to normaliser

### PR DESCRIPTION
**Pull request by Dragos0000**

### CODE REVIEW

File: ted_sws/metadata_normaliser/services/metadata_normalizer.py [hunk 1]
```diff
@@ -132,12 +162,18 @@ def get_form_type_and_notice_type(cls, ef_map: pd.DataFrame, sf_map: pd.DataFram
...
-                                             document_code=document_type_code)
+        filter_variables = cls.get_filter_variables_values(form_number=form_number, filter_map=filter_map,
+                                                           extracted_notice_type=extracted_notice_type,
+                                                           legal_basis=legal_basis,
+                                                           document_type_code=document_type_code)
+        filtered_df = filter_df_by_variables(df=mapping_df, form_number=filter_variables["form_number"],
+                                             sf_notice_type=filter_variables["sf_notice_type"],
+                                             legal_basis=filter_variables["legal_basis"],
+                                             document_code=filter_variables["document_code"])
         form_type = filtered_df["form_type"].values[0]
```
Comment (CaptainOfHacks) [inline ted_sws/metadata_normaliser/services/metadata_normalizer.py:173]: use constants instead raw strings

File: tests/unit/metadata_normaliser/test_metadata_normaliser.py [hunk 2]
```diff
@@ -73,11 +73,28 @@ def test_filter_df_by_variables():
...
+    extracted_metadata = XMLManifestationMetadataExtractor(xml_manifestation=raw_notice.xml_manifestation).to_metadata()
+    extracted_metadata_normaliser = ExtractedMetadataNormaliser(extracted_metadata=extracted_metadata)
+    filter_map = MappingFilesRegistry().filter_map_df
+    filter_variables_dict = extracted_metadata_normaliser.get_filter_variables_values(form_number="F03",
+                                                                                      filter_map=filter_map,
+                                                                                      extracted_notice_type=None,
+                                                                                      document_type_code="7",
+                                                                                      legal_basis="legal")
+    assert isinstance(filter_variables_dict,dict)
+    assert filter_variables_dict["form_number"] == "F03"
```
Comment (CaptainOfHacks) [inline tests/unit/metadata_normaliser/test_metadata_normaliser.py:97]: use constants instead raw strings

**Review by CaptainOfHacks — APPROVED** [review]
Well done!

File: ted_sws/resources/mapping_files/sforms_mapping.csv [hunk 3]
```diff
@@ -1,102 +1,44 @@
...
+F01,PRI_ONLY,,,4
+F01,PRI_REDUCING_TIME_LIMITS,,,7
+F02,,,,16
+F03,,,,29
+F04,PER_CALL_COMPETITION,,,11
+F04,PER_ONLY,,,5
+F04,PER_REDUCING_TIME_LIMITS,,,8
+F05,,,,17
+F06,,,,30
+F07,,,O,15
```
Comment (costezki) [inline ted_sws/resources/mapping_files/sforms_mapping.csv:12]: did we forget a `0` there ?

File: ted_sws/resources/mapping_files_registry.py [hunk 4]
```diff
@@ -37,3 +37,6 @@ def sf_notice_df(self):
     def ef_notice_df(self):
         return get_mapping_csv_file("eforms_mapping.csv")
 
+    @property
+    def filter_map_df(self):
+        return get_mapping_csv_file("df_filter_map.csv")
```
Comment (costezki) [inline ted_sws/resources/mapping_files_registry.py:42]: the filename shall be a constant (light-coded), not a hardcoded value

Comment (costezki) [reply ted_sws/resources/mapping_files_registry.py:42]: idem for other mappings

**Review by costezki — APPROVED** [review]
well done! 

do not hard-coded so HARD the values which can be constants. 

there is a typo (possibly) in the CSV

**Pull request merged**